src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -488,6 +494,7 @@ class LicenceService(
         activatedLicences,
         "Licence automatically deactivated as the approved licence version was activated",
       )
+      inactivateTimedOutLicenceVersions(activatedLicences, "Licence automatically deactivated as the approved licence version was activated")
```

Comment:
Called as part of the activation job for IS-91 licences and other licences where the release event failed.